frontend/src/views/Game.tsx

```diff
@@ -158,22 +179,26 @@ function GamePage() {
 
     // Get the result of promises and set the default code list.
     Promise.all(promises).then((result) => {
-      const codeMap = result[0];
+      const codeList = [];
+
+      for (let i = 0; i < result.length; i += 1) {
+        codeList.push(result[i][language[i]]);
+      }
 
       // If previous code and language specified, save those as defaults
       if (code) {
-        codeMap[language] = code;
+        codeList[currentProblem.valueOf()] = code;
```

Comment:
so this only works with one set of code for now right? totally fine if so, just wanted to double check though

Reply:
Yep, it's a much bigger change to keep track of all code that was previously submitted because we'd half to keep track of a lot more on the back end per user.